repo/packages/H/hdfs/0/command.json

```diff
@@ -1,5 +1,5 @@
 {
   "pip": [
-    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.0#dcos-hdfs=0.1.0"
+    "git+https://github.com/mesosphere/dcos-hdfs.git@0.1.1#dcos-hdfs=0.1.1"
```

Comment:
Ok, discussed with others and for passing the config, the proper fix is https://mesosphere.atlassian.net/browse/HDFS-25. I closed HDFS-85 for this reason.

The main issue that @jsancio was referring to was a team city build and an automated update to an s3 bucket for the subcommand code: git+https://github.com/mesosphere/dcos-hdfs.git@0.1.1#dcos-hdfs=0.1.1. Since automated builds are important and hdfs itself is now built in team city, could you do this?